onnx/defs/logical/defs.cc

```diff
@@ -86,7 +86,11 @@ ONNX_OPERATOR_SET_SCHEMA(
         .FillUsing(BinaryLogicDocGenerator("greater"))
         .TypeConstraint(
             "T",
-            {"tensor(float16)", "tensor(float)", "tensor(double)"},
+            {"tensor(int32)",
```

Comment:
thank you Lu! shall we add all numeric types please?

Reply:
Sure, I will do that and bump up the opset version too